src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -408,4 +408,15 @@ public ScmPolling pollScm() {
 
         return links;
     }
+
+    /**
+     * Deletes the current job
+     */
+    public void delete() {
+        this.open();
+        clickLink("Delete Project");
```

Comment:
Or it could change in the future.  Maybe parametertize it out or override with a delete(String label) method?